pkg/rancher-desktop/main/extensions/extensions.ts

```diff
@@ -207,11 +273,44 @@ export class ExtensionImpl implements Extension {
     return true;
   }
 
+  protected async uninstallContainers(metadata: ExtensionMetadata) {
+    function isImage(input: any): input is { image: string } {
```

Comment:
There appears to be some duplication with `isImage` and `isComposefile` between `installContainers` and `uninstallContainers`. Can we make these functions methods of `ExtensionImpl`, or reduce the repetition in some other way?

Reply:
Good idea. I just made it a (file-scoped) global function.